src/libraries/System.Net.Http/src/System/Net/Http/HttpTelemetry.cs

```diff
@@ -90,16 +90,23 @@ private void RequestFailed(string exceptionMessage)
             WriteEvent(eventId: 3, exceptionMessage);
         }
 
+        [NonEvent]
+        private void ConnectionEstablished(byte versionMajor, byte versionMinor, long connectionId, Uri uri, IPEndPoint? remoteEndPoint)
+        {
+            string? remoteAddress = remoteEndPoint?.Address?.ToString();
+            ConnectionEstablished(versionMajor, versionMinor, connectionId, uri.Scheme, uri.Host, uri.Port, remoteAddress);
+        }
+
         [Event(4, Level = EventLevel.Informational)]
```

Comment:
```suggestion
        [Event(4, Level = EventLevel.Informational, Version = 1)]
```

Same with the other events where the signature changed